File: source/code/java/authentication/login_attempt_req.java
try {
    AuthenticationRequest authenticationRequest = UsernamePasswordRequest
        .builder()
        .setUsernameOrEmail("han@newrepublic.gov")
        .setPassword("first2shoot!")
        .build();
    AuthenticationResult result = application.authenticateAccount(authenticationRequest);

    Account account = result.getAccount();
} catch(IncorrectCredentialsException e) {
    e.printStackTrace();
}

File: source/code/nodejs/authentication/login_attempt_req.js
var applicationHref = 'https://api.stormpath.com/v1/applications/$YOUR_APPLICATION_ID';

var authRequest = {
  username: 'han@newrepublic.gov',
  password: 'first2shoot!'
};

client.getApplication(applicationHref, function (err, application) {
  if (err) {
    return console.error(err);
  }

  application.authenticateAccount(authRequest, function (err, authResult) {
    if (err) {
      return console.error(err);
    }

    authResult.getAccount(function (err, account) {
      if (err) {
        return console.error(err);
      }

      console.log('Account has authenticated: ', account);
    });
  });
});